artcommon/artcommonlib/konflux/konflux_db.py

```diff
@@ -210,7 +210,7 @@ async def get_latest_build(
             name: str,
             group: str,
             outcome: KonfluxBuildOutcome = KonfluxBuildOutcome.SUCCESS,
-            assembly: str = 'stream',
+            assembly: str = None,
```

Comment:
```suggestion
            assembly: typing.Optional[str] = None,
```

`str` doesn't actually accept None value and will cause type checking error if you have.